BasicAndroidAccessibility/app/src/main/java/com/example/android/basicandroidaccessibility/HomeFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.android.basicandroidaccessibility
+
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.navigation.fragment.findNavController
+
+
+class HomeFragment : Fragment() {
+
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_home, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+
+        val buttonContentLabeling = view.findViewById<Button>(R.id.content_labeling_button)
+        buttonContentLabeling?.setOnClickListener {
+            findNavController().navigate(R.id.action_homeFragment_to_contentLabelingFragment, null)
```

Comment:
Use convenience method of generating navigation click listeners:
https://developer.android.com/reference/kotlin/androidx/navigation/Navigation.html#createNavigateOnClickListener(kotlin.Int)